.github/ISSUE_TEMPLATE/release.md

```diff
@@ -17,58 +17,80 @@ assignees: ''
   - [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/)
 - [ ] **Re-certify for Red Hat**: If a new major version of Red Hat Enterprise Linux is released (e.g., v10 after v9), re-certify Orchard Core. Refer to:
   - Orchard's [Red Hat Ecosystem Catalog profile](https://catalog.redhat.com/software/applications/detail/223797) for the last certified version.
   - [Red Hat Customer Portal](https://access.redhat.com/articles/3078) for the latest version details.
   - [Certification guide](https://docs.orchardcore.net/en/latest/topics/red-hat-ecosystem-catalog-certification/) for certification steps.
+- [ ] If everything looks good and all checks pass, merge the version PR.
 
 ## Step 5: Creating the New Release
 
 1. Navigate to the [GitHub Releases page](https://github.com/OrchardCMS/OrchardCore/releases/new).
 2. In the "**Choose a tag**" menu, enter the new version number (e.g., `v3.0.0`) and select "**+ Create tag: v... on publish**."
 3. Change the target branch from `main` to your release branch (e.g., `release/3.0`).
 4. Enter the version number in the Title field (e.g., `3.0.0`).
-5. Click **Generate release notes** to automatically create release notes based on the changes.
-6. Ensure the "Set as the latest release" checkbox is checked, then click **Publish release**.
+5. Click **Generate release notes** to automatically create release notes based  on the changes.
+6. Add a link to the release notes on the docs site: `Check out the full release notes [here](https://docs.orchardcore.net/en/latest/releases/3.0.0/).`
+7. Ensure the "Set as the latest release" checkbox is checked, then click **Publish release**.
+
+## Step 6: Aligning Branches
+
+**Merge to `main`**: After releasing the new version, merge the release branch into the main branch to ensure `main` contains all administrative changes.
+
+- [ ] Create a pull request from the release branch (e.g., `release/3.0`) into `main`.
+- [ ] Resolve any merge conflicts using external tools (e.g., Fork) to avoid auto-merging `main` into the release branch. **Important**: DO NOT resolve conflicts using GitHub's interface; that will automatically merge `main` into the release branch, which must be avoided.
+- [ ] Merge the PR if all checks pass. If there are merge conflicts, then you'll need to merge to `main` manually using the following steps:
+  1. Fetch the latest changes from the Git repository.
+  2. Checkout the `main` branch.
+  3. Merge the release branch (e.g., `release/3.0`) into `main` with a merge commit (NOT a squash merge). Use the commit message pattern `Merge release/3.0.0 to main`.
+  4. Resolve any conflicts.
+  5. Push the changes to `main`. This action requires a user with the ability to force-push into `main`, as it is protected by default.
+  6. GitHub will automatically delete the release branch; go back to the new merged PR to restore it.
 
-## Step 6: Post-Release Tasks
+## Step 7: Post-Release Tasks
```

Comment:
If you like the ing, the would Be Posting Release Tasks but I don't like the ing :)

Reply:
@Piedone this one?

Reply:
Not sure what you mean, sorry. If you mean this I left out: isn't this answered by https://github.com/OrchardCMS/OrchardCore/pull/17499#discussion_r1964504015?